lighthouse-core/report/html/report-template.html

```diff
@@ -47,6 +47,17 @@
       // is in the document.
       const features = new ReportUIFeatures(dom);
       features.initFeatures(window.__LIGHTHOUSE_JSON__);
+
+      // TODO(cjamcl) - Persist dark mode state.
+      document.addEventListener('keyup', e => {
```

Comment:
is this a real thing we think we'll keep in the report? :D

Reply:
I'd like to. May need to accompany it with a notice "press d to ..." so it can be undone when activated on accident.